### PR TITLE
3.0.0 changelog

### PR DESCRIPTION
**Pull request by travissanderson-wf**

### CODE REVIEW

**Comment by aviary2-wf** [comment]
## Security Insights
No security relevant content was detected by automated scans.

## Action Items
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://wiki.atl.workiva.net/pages/viewpage.action?pageId=42762752#SecurityInsightFramework&Tooling-ConfiguringAviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+## [3.0.0](https://github.com/Workiva/r_tree/compare/2.1.6...3.0.0)
+ - Removed deprecated symbols from the public API
+ - To migrate, simply to switch from using `rtree.insert` or `rtree.load` to `rtree.add`
```
Comment (kerbyseeley-wf) [inline CHANGELOG.md:3]: Should this be "To migrate, simply switch..."

**Review by kerbyseeley-wf — APPROVED** [review]
+10

**Comment by travissanderson-wf** [comment]
@Workiva/release-management-p

**Review by rmconsole-wf — APPROVED** [review]
+1 from RM

**Pull request merged**